src/auth-service/utils/request-access.js

```diff
@@ -403,32 +382,12 @@ const requestAccess = {
 
   delete: async (tenant, filter) => {
     try {
-      let responseFromRemoveCandidate = await CandidateModel(
+      const responseFromRemoveCandidate = await CandidateModel(
         tenant.toLowerCase()
       ).remove({
         filter,
       });
-
-      if (responseFromRemoveCandidate.success == true) {
```

Comment:
just simple refactoring to reduce on lines of code and also align all the error responses to the AirQo standard of JSON response =`{errors: {message:"error message"}}`